test/suites/chopsticks/test-upgrade-chain.ts

```diff
@@ -95,14 +95,30 @@ describeSuite({
 
     it({
       id: "T2",
-      timeout: 60000,
+      timeout: 900_000, // long timeout required for multi-migrations to complete
       title: "Can send balance transfers",
-      test: async () => {
-        const balanceBefore = (await api.query.system.account(DUMMY_ACCOUNT)).data.free.toBigInt();
-        await api.tx.balances.transferAllowDeath(DUMMY_ACCOUNT, parseEther("1")).signAndSend(alith);
-        await context.createBlock({ count: 2 });
-        const balanceAfter = (await api.query.system.account(DUMMY_ACCOUNT)).data.free.toBigInt();
-        expect(balanceBefore < balanceAfter).to.be.true;
+      test: async function () {
+        const specName = api.consts.system.version.specName.toString();
+        // Multi-migration take 15 minutes to complete on moonbeam,
+        // and there is already another test (C03) taht test the migration itself.
+        // So we skip this test otherwise we need 30 minutes to complete C01 + C03.
+        if (specName !== "moonbeam") {
```

Comment:
Do we still need this after https://github.com/moonbeam-foundation/moonbeam/pull/3550/commits/35be08c79a6ff0d33734daed341926f8d6f24b77?